next/modules/common/ImageGallery/ImageGallery.tsx

```diff
@@ -0,0 +1,199 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/interactive-supports-focus */
+
+import MImage from '@components/Atoms/MImage'
+import { UploadImageEntityFragment } from '@services/graphql'
+import { onEnterOrSpaceKeyDown } from '@utils/onEnterOrSpaceKeyDown'
+import cx from 'classnames'
+import { useTranslation } from 'next-i18next'
+import { useCallback, useMemo, useState } from 'react'
+import { useResizeDetector } from 'react-resize-detector'
+import { useOverlayTriggerState } from 'react-stately'
+
+import ImageLightBox from './ImageLightBox'
+
+// copied from marianum https://github.com/bratislava/marianum.sk/blob/master/next/components/molecules/ImageGallery.tsx
+
+export type ImageGalleryProps = {
+  images: UploadImageEntityFragment[]
+  variant?: 'below' | 'aside'
```

Comment:
I would maybe add a comment tell future us that "aside" is used for header and "below" is used in page content/dynamic sections.